apps/web-giddh/src/app/reports/components/sales-register-component/sales.register.component.html

```diff
@@ -1,119 +1,120 @@
-                    <li>
-                        <div class="select-style width-200 select-year">
-                            <select name="">
-                                <option value="">August 2017</option>
-                                <option value="">Weekly</option>
-                                <option value="">Custom Range</option>
-                            </select>
+                                <li>
+                                    <a href="javascript:void(0)" class="dropdown-item">
+                                        <input type="checkbox">
+                                        Discount</a>
+                                </li>
+                                <li>
+                                    <a href="javascript:void(0)" class="dropdown-item">
+                                        <input type="checkbox">
+                                        Tax</a>
+                                </li>
+                                <li>
+                                    <a href="javascript:void(0)" class="dropdown-item">
+                                        <input type="checkbox">
+                                        Round Off</a>
+                                </li>
+                                <li>
+                                    <a href="javascript:void(0)" class="dropdown-item">
+                                        <input type="checkbox">
+                                        Tax on Freight</a>
+                                </li>
+                                <li>
+                                    <a href="javascript:void(0)" class="dropdown-item">
+                                        <input type="checkbox">
+                                        Freight</a>
+                                </li>
+                                <li>
+                                    <a href="javascript:void(0)" class="dropdown-item">
+                                        <input type="checkbox">
+                                        Net Sales</a>
+                                </li>
+                            </ul>
                         </div>
-                    </li>
-
-                    <li>
-                        <div class="select-style width-200">
-                            <select name="">
-                                <option value="">Monthly</option>
-                                <option value="">Weekly</option>
-                                <option value="">Custom Range</option>
-                            </select>
-                        </div>
-                    </li>
-                    <li>
-                        <a href="javascript:void(0)" class="exand-icon d-inline-block middle"><i
-                                class="icon-expand"></i></a>
-                    </li>
-                </ul>
-
-
+                    </div>
+                </div>
+            </div>
 
+            <div class="col-xs-6 text-right">
```

Comment:
add col-6 or col-sm-6 in place of col-xs-6 @shalinee-walkover

Reply:
Done